x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/configuration_step_form.tsx

```diff
@@ -445,6 +456,26 @@ export const ConfigurationStepForm: FC<CreateAnalyticsStepProps> = ({
       >
         <Fragment />
       </EuiFormRow>
+      {unsupportedRuntimeFields && unsupportedRuntimeFields.length > 0 && (
+        <>
+          <EuiCallOut size="s" color="warning">
+            <FormattedMessage
+              id="xpack.ml.dataframe.analytics.create.unsupportedRuntimeFields"
+              defaultMessage="The runtime fields {unsupportedRuntimeFields} are not supported for analysis."
```

Comment:
In case only 1 runtime field is unsupported, the sentence should be alternated.

Reply:
Updated here https://github.com/elastic/kibana/pull/90363/commits/8adf58ffa864c8fab43de889b13b3c001139ddbe